core/src/main/java/kafka/server/NetworkUtils.java

```diff
@@ -82,7 +82,7 @@ public static NetworkClient buildNetworkClient(String prefix,
             config.connectionSetupTimeoutMs(),
             config.connectionSetupTimeoutMaxMs(),
             time,
-            false,
+            true,
```

Comment:
This one is only used by the AddPartitionsToTxnManager, right?

Reply:
Yes, I believe this is the only usage now.